liquibase-standard/src/main/java/liquibase/analytics/AnalyticsConfiguration.java

```diff
@@ -0,0 +1,22 @@
+package liquibase.analytics;
+
+import liquibase.configuration.AutoloadedConfigurations;
+import liquibase.configuration.ConfigurationDefinition;
+
+public class AnalyticsConfiguration implements AutoloadedConfigurations {
```

Comment:
This class is largely temporary and will either be completely removed or changed significantly.